data/maps/AquaHideout_B1F/scripts.inc

```diff
@@ -1,6 +1,6 @@
 AquaHideout_B1F_MapScripts:: @ 82339A7
 	map_script MAP_SCRIPT_ON_RESUME, AquaHideout_B1F_MapScript1_2339B2
-	map_script MAP_SCRIPT_ON_TRANSITION, AquaHideout_B1F_MapScript1_2339D0
+	map_script MAP_SCRIPT_ON_TRANSITION, AquaHideout_B1F_MapScript1_OnTransition
```

Comment:
None of the `OnTransition` labels should have `MapScript1` in them.  That's redundant.